libafl/src/fuzzer/mod.rs

```diff
@@ -682,6 +697,19 @@ where
         )?;
         Ok(idx)
     }
+
+    /// Adds an input, even if it's not considered `interesting` by any of the executors
```

Comment:
This comment seems wrong?

Reply:
Oh, no this one is just "copied" from above, just git diff algo showing this function as "new". Also note that this is true, because inputs are always added, even if not interesting